FinniversKit.xcodeproj/project.pbxproj

```diff
@@ -2030,18 +2037,19 @@
 		CF8F021421DCC5B2004C4042 /* Components */ = {
 			isa = PBXGroup;
 			children = (
-				CFB61377229ACB6200DE283D /* IconCollection */,
 				083349EA21EF24AE000E68D3 /* BottomSheet */,
 				CF8F021D21DCC691004C4042 /* Broadcast */,
 				CF8F021E21DCC6A0004C4042 /* Button */,
 				CFCDDA5D22731D4D005E1F4A /* Callout */,
+				DAE738C522C4C84C001C69E3 /* CarouselVIew */,
```

Comment:
That capital `I` is a bit out of place there I think 😄

Reply:
Haha, I fix